<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>324</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LabStick Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Readings</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>235</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="myTableView"/>
       </item>
       <item>
        <widget class="QPushButton" name="rescan_pushButton">
         <property name="text">
          <string>Rescan Devices</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Temperature Unit</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>103</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="config_celcius_radioButton">
          <property name="text">
           <string>Celcius</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="config_kelvin_radioButton">
          <property name="text">
           <string>Kelvin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="config_fahr_radioButton">
          <property name="text">
           <string>Fahrenheit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Thermocouple</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>24</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Advanced</string>
      </property>
      <widget class="QCheckBox" name="config_showADC_checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Show ADC readings</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="config_logADC_checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Log ADC readings</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="config_logColdJunction_checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Log Cold Junction readings</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Graph</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>241</width>
         <height>60</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Moving avg filter length:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="config_movAvg_spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Plot history (units)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="config_movAvg_spinBox_2">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Indication Led</string>
      </property>
      <widget class="QComboBox" name="indicationLed_comboBox">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>30</y>
         <width>151</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Logging</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Device Settings</string>
      </property>
      <widget class="QRadioButton" name="logTab_radioButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>25</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Log them all</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="logTab_radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>54</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Log one device</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="logTab_comboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>131</x>
         <y>51</y>
         <width>104</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="logTab_pushButton">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>190</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Logging</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="logTab_lcdNumber">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="value" stdset="0">
       <double>12.000000000000000</double>
      </property>
     </widget>
     <widget class="QTextEdit" name="logTab_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>431</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Real-time graph</string>
     </attribute>
     <widget class="QCustomPlot" name="myPlot" native="true">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>8</y>
        <width>441</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>451</width>
        <height>49</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current Device ID for Plotting:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="graphTab_comboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="logTab_statusLabel">
         <property name="text">
          <string>Status: Idle ...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="logTab_lastReadLabel">
         <property name="text">
          <string>Last Read: ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>./qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
